CONTRIBUTING.md

```diff
@@ -3,4 +3,4 @@
 In favor of active development we accept contributions from everyone. You can contribute by submitting a bug, creating pull requests or even by improving documentation.
 
 Below is the guide to be followed strictly before submitting your pull requests.
-http://adonisjs.com/docs/3.1/contributing
+https://adonisjs.com/docs/4.1/contribution-guide
```

Comment:
It's better to remove the version from the link.

```suggestion
https://adonisjs.com/docs/contribution-guide
```

Reply:
Sure, please wait.

Reply:
Done, thank you for your feedback.